src/util.h

```diff
@@ -133,6 +131,16 @@ inline bool IsSwitchChar(char c)
 #endif
 }
 
+class ArgsManager
+{
+protected:
+    CCriticalSection cs_args;
+    std::map<std::string, std::string> mapArgs;
+    std::map<std::string, std::vector<std::string> > mapMultiArgs;
```

Comment:
No real gain, just slightly simpler (because multi_map is intended to be used I think exactly in this type of case).